language/move-prover/src/prelude.bpl

```diff
@@ -541,7 +541,7 @@ procedure {:inline 1} BorrowField(src: Reference, f: FieldName) returns (dst: Re
 
     p := p#Reference(src);
     size := size#Path(p);
-	p := Path(p#Path(p)[size := f], size+1);
+        p := Path(p#Path(p)[size := f], size+1);
```

Comment:
Is this perhaps a tab character? If so please avoid tabs because they are showing differently here and there. Most editors should have settings to have the same benefits without the tab character,